### PR TITLE
fix(windrose): Fix prevailing wind direction getter.

### PR DESCRIPTION
**Pull request by saeranv**
@chriswmackey 

Here's the fix. I should have made sure to write a test for it the first time.

### CODE REVIEW

**Comment by saeranv** [comment]
Fascinating, it looks like in Python 2.7 the dictionary iterables are in the opposite order of Python 3, which is causing a test to fail. Will change and recommit.

File: ladybug/windrose.py [hunk 1]
```diff
@@ -326,15 +326,33 @@ def histogram_data(self):
...
-            self._prevailing_direction = dirvals[dirvals.index(max(dirvals))]
-        return self._prevailing_direction % 360.0
+            # group by frequency
+            groups = {}
+            for dirval in self.direction_values:
+                dirval = int(dirval)
+                if dirval not in groups:
+                    groups[dirval] = 1
+                else:
+                    groups[dirval] += 1
```
Comment (chriswmackey) [inline ladybug/windrose.py:340]: I feel like this is really going to backfire on us if we ever have an EPW where the wind direction doesn't fall into 10-degree increments. Can we use the binned `_histogram_data` and the user-input `number_of_directions` for this case instead?

File: ladybug/windrose.py [hunk 2]
```diff
@@ -326,15 +326,33 @@ def histogram_data(self):
...
+            max_freq = 0
+            for dirv, freq in groups.items():
+                if max_freq < freq:
+                    max_dirv = [dirv]  # Reset list
+                    max_freq = freq
+                elif max_freq == freq:
+                    max_dirv.append(dirv)
+
+            # set tuple
+            self._prevailing_direction = tuple(max_dirv)
```
Comment (chriswmackey) [inline ladybug/windrose.py:353]: Also, why is this a tuple? Shouldn't it be a single value for the direction in degrees?

Comment (saeranv) [reply ladybug/windrose.py:353]: That's to account for the edge case of ties (same frequency of values in two or more directions).

**Comment by saeranv** [comment]
@chriswmackey  This change will return the direction based on the number of directions of the windrose.

**Pull request merged**